### ISSUE TITLE
What about seeding?

### ISSUE THREAD

**Issue by akaleeroy** [sayem314/torrenter#2]
Maybe make a note in the README that this does not allow seeding after the download completed.

**Comment by sayem314** [comment]
Yes, you are right, torrenter is meant for newbies so I don't mess with seeding yet. Maybe torrenter gui version (planned) will seed like any other torrent clients.
